Documentation/operations/performance/scalability/identity-relevant-labels.rst

```diff
@@ -24,20 +24,20 @@ following exceptions:
 =================================================== =========================================================
 Label                                               Description
 --------------------------------------------------- ---------------------------------------------------------
-``any:!io.kubernetes``                              Ignore all ``io.kubernetes`` labels
-``any:!kubernetes\.io``                             Ignore all other ``kubernetes.io`` labels
-``any:!statefulset\.kubernetes\.io/pod-name``       Ignore ``statefulset.kubernetes.io/pod-name`` label
-``any:!apps\.kubernetes\.io/pod-index``             Ignore ``apps.kubernetes.io/pod-index`` label
-``any:!batch\.kubernetes\.io/job-completion-index`` Ignore ``batch.kubernetes.io/job-completion-index`` label
-``any:!batch\.kubernetes\.io/controller-uid``       Ignore ``batch.kubernetes.io/controller-uid`` label
-``any:!beta\.kubernetes\.io``                       Ignore all ``beta.kubernetes.io`` labels
-``any:!k8s\.io``                                    Ignore all ``k8s.io`` labels
-``any:!pod-template-generation``                    Ignore all ``pod-template-generation`` labels
-``any:!pod-template-hash``                          Ignore all ``pod-template-hash`` labels
-``any:!controller-revision-hash``                   Ignore all ``controller-revision-hash`` labels
-``any:!annotation.*``                               Ignore all ``annotation`` labels
-``any:!controller-uid``                             Ignore all ``controller-uid`` labels
-``any:!etcd_node``                                  Ignore all ``etcd_node`` labels
+``!io.kubernetes``                                  Ignore all ``io.kubernetes`` labels
```

Comment:
While we're here, let's escape this extra `.`:

```suggestion
``!io\.kubernetes``                                 Ignore all ``io.kubernetes`` labels
```

Reply:
will change in next commit